test/src/test/java/hudson/triggers/TriggerTest.java

```diff
@@ -0,0 +1,90 @@
+package hudson.triggers;
+
+/*
+ * The MIT License
+ *
+ * Copyright (c) 2016 Felix Belzunce Arcos
```

Comment:
@jtnord i also tried to suggest to use correct emails to differentiate, but nobody cares ;)